01-node-tutorial/answers/03-modules.js

```diff
@@ -0,0 +1,13 @@
+// CommonJS, every file is module (by default)
+// Modules - Encapsulated Code (only share minimum)
+const names = require('./04-names')
+const sayHi = require('./05-utils')
+const data = require('./06-alternative-flavor')
+require('./07-mind-grenade')
+sayHi('susan')
+sayHi(names.john)
+sayHi(names.peter)
+ 
+sayHi(names.anna)
+console.log(data.items.bob)
+sayHi(data.items.nik)
```

Comment:
Good job!
You probably meant `persons` instead of `items` here.

```suggestion
console.log(data.persons.bob)
sayHi(data.persons.nik)
```

Reply:
you are right. Thank you